src/corelibs/U2View/src/ov_msa/MaEditor.cpp

```diff
@@ -137,15 +138,11 @@ MaEditor::MaEditor(GObjectViewFactoryId factoryId, const QString& viewName, Mult
     gotoSelectedReadAction->setEnabled(false);
     connect(gotoSelectedReadAction, &QAction::triggered, this, &MaEditor::sl_gotoSelectedRead);
 
-    showOverviewAction = new QAction(QIcon(":/core/images/msa_show_overview.png"), tr("Overview"), this);
-    showOverviewAction->setObjectName("Show overview");
-    showOverviewAction->setCheckable(true);
-    showOverviewAction->setChecked(true);
-
-    clearSelectionAction = new QAction(tr("Clear selection"), this);
-    clearSelectionAction->setShortcut(Qt::Key_Escape);
-    clearSelectionAction->setShortcutContext(Qt::WidgetWithChildrenShortcut);
-    connect(clearSelectionAction, &QAction::triggered, this, &MaEditor::sl_onClearActionTriggered);
+    multilineViewAction = new QAction(QIcon(":core/images/multiline_view.png"), tr("Multiline View"), this);
+    multilineViewAction->setObjectName("Multiline View");
```

Comment:
Use either snake_case (used in other places in UGENE) or camelCase (default for QT) for object names.

Reply:
?
can't understand
The word `multiline` is a single word here, same as `goto`
See:
```
gotoSelectedReadAction->setEnabled(false);
```

Reply:
I mean setObjectName("Multiline View");

Reply:
see now